src/command.rs

```diff
@@ -61,7 +63,18 @@ impl SubstrateCli for Cli {
 
 			// `.with_genesis_config(Default::default)` won't work, but should.
 			let tmp = sc_chain_spec::GenesisConfigBuilderRuntimeCaller::<'_, ()>::new(&code);
-			let genesis = tmp.get_default_config()?;
+			let mut genesis = tmp.get_default_config()?;
```

Comment:
The right way to do this nowadays and in a way that keeps a node a true omni-node is to use `chain-spec-builder`, specifically the `patch` and `presets` feature :)

Reply:
It would be great if you explore that and add some comments about how to do it in the README.

Reply:
Similar approach should be taken in the FRAME template and assignment.

Reply:
See https://github.com/kianenigma/frameless-runtime-sg-2024's README

Reply:
> See https://github.com/kianenigma/frameless-runtime-sg-2024 's README

Not able to access it. Could you add me please?